### PR TITLE
add cmake flag IREE_HAL_VULKAN_FEATURE_ENABLE_DEBUG_UTILS

### PR DESCRIPTION
**Pull request by PhaneeshB**

### CODE REVIEW

File: build_tools/python_deploy/build_linux_packages.sh [hunk 1]
```diff
@@ -129,6 +129,7 @@ function build_iree_runtime() {
 function build_iree_runtime_instrumented() {
   IREE_HAL_DRIVER_CUDA=ON IREE_BUILD_TRACY=ON IREE_ENABLE_RUNTIME_TRACING=ON \
   IREE_RUNTIME_CUSTOM_PACKAGE_SUFFIX="-instrumented" \
+  IREE_HAL_VULKAN_FEATURE_ENABLE_DEBUG_UTILS=2 \
```
Comment (powderluv) [inline build_tools/python_deploy/build_linux_packages.sh]: this should be `ON`

File: compiler/setup.py [hunk 2]
```diff
@@ -232,6 +232,7 @@ def prepare_installation():
         "-DCMAKE_BUILD_TYPE={}".format(cfg),
         get_env_cmake_option("IREE_TARGET_BACKEND_CUDA"),
         get_env_cmake_option("IREE_ENABLE_CPUINFO", "ON"),
+        get_env_cmake_option("IREE_HAL_VULKAN_FEATURE_ENABLE_DEBUG_UTILS"),
```
Comment (powderluv) [inline compiler/setup.py:235]: this actually goes into the runtime/setup.py (not compiler/setup.py)

File: build_tools/python_deploy/build_linux_packages.sh (hunk 1 above)
Comment (PhaneeshB) [reply build_tools/python_deploy/build_linux_packages.sh]: Okay

Comment (PhaneeshB) [reply build_tools/python_deploy/build_linux_packages.sh]: Changed the value to 2 
based on this 
https://github.com/nod-ai/SHARK-Runtime/blob/shark/runtime/src/iree/hal/drivers/vulkan/api.h#L39

will change it to ON

**Comment by PhaneeshB** [comment]
closing PR as we don't need it anymore

**Pull request closed without merging**